cmd/kubelet/app/options/options.go

```diff
@@ -96,6 +96,9 @@ func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
 	fs.Var(componentconfig.IPVar{Val: &s.Address}, "address", "The IP address for the Kubelet to serve on (set to 0.0.0.0 for all interfaces)")
 	fs.Int32Var(&s.Port, "port", s.Port, "The port for the Kubelet to serve on.")
 	fs.Int32Var(&s.ReadOnlyPort, "read-only-port", s.ReadOnlyPort, "The read-only port for the Kubelet to serve on with no authentication/authorization (set to 0 to disable)")
+	fs.BoolVar(&s.TLSCertAuth, "tls-cert-auth", s.TLSCertAuth, "Enable TLS client authentication")
```

Comment:
I don't think a fixed group here works. This is essentially introducing authz to the kubelet, and there are at least a few different levels of access needed:

Read only access to metrics
Read only access to logs
Full access to the whole kubelet API 

We'll likely end up wanting webhook authz so authz policy can be specified in one place. I think the authz limitation should be a follow up to this